warrior/WarriorCore/Classes/kw_driver_class.py

```diff
@@ -201,7 +201,6 @@ def get_argument_as_keywords(self):
     def execute_method_for_keyword(self):
         """Executes a method corresponding to keyword """
 
-        Utils.config_Utils.set_datarepository(self.data_repository)
```

Comment:
The same line is added in testcase_driver in L40
It gives reference to data_repo between tc_driver and kw_driver_class
and since we can't call step from outside, tc will be the smallest file warrior can execute from